src/popup.html

```diff
@@ -5,49 +5,41 @@
     <base href="/">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="css/bootstrap.min.css">
     <link rel="stylesheet" href="css/popup.css">
     <script defer type="text/javascript" src="scripts/popup.js"></script>
 </head>
 <body class="text-center">
     <div id="horn" class="popup-link bg-warning">
-        Sound Horn: <span id="huntTimer">-:--</span>
+        <span id="huntTimer"></span>
     </div>
-    <div class="container-fluid">
+    <main>
         <h3><a href="https://www.mhct.win" target="_blank" class="popup-link-out">MH Community Tools</a></h3>
-        <span id="version" class="text-muted" style="font-size:small;"></span>
         <table class="table text-center table-responsive table-hover table-bordered">
         <tbody>
             <tr><td id="userhistory" class="popup-link">
                 Your Hunt History
             </td></tr>
             <tr><td id="mhmh" class="popup-link">
-                MHCT Map Helper (auto)
+                Map Helper
             </td></tr>
             <tr><td>
-                <a href="https://www.mhct.win/attractions.php" target="_blank" class="popup-link-out">Mouse Attraction Rates</a>
+                <a href="https://www.mhct.win/attractions.php" target="_blank" class="popup-link-out">Attraction Rates</a>
             </td></tr>
             <tr><td>
-                <a href="https://www.mhct.win/loot.php" target="_blank" class="popup-link-out">Looter (Drop Rates)</a>
```

Comment:
why are we moving it down? Looter/drop rates is one of the most used tools if not the most used.

Reply:
👍  Reverted